packages/flutter_test/lib/src/binding.dart

```diff
@@ -106,6 +111,32 @@ mixin TestDefaultBinaryMessengerBinding on BindingBase, ServicesBinding {
   }
 }
 
+/// An accessibility announcement made by the Flutter framework.
+///
+/// This class is mostly used by the testing api to store the announcements
+/// in a user-friendly structure so that the code consumers can check the details
```

Comment:
nit: hopefully all our API is user-friendly 😄 Instead, I'd just say "in a structured form"
nit: "code consumer" sounds confusing. Maybe say "so that tests can verify announcement details"?

Reply:
Thanks for the suggestions, Done!